### ISSUE TITLE
Tag latest version

### ISSUE THREAD

**Issue by robertdj** [JuliaGeometry/VoronoiDelaunay.jl#18]
Is it possible to tag the lastest version of this package and GeometricalPredicates? It would be nice to have a number for a `REQUIRE` file :-)

**Comment by tkelman** [comment]
cc @skariel? or maybe @simondanisch? Also see https://github.com/JuliaGeometry/VoronoiDelaunay.jl/pull/20, the `REQUIRE` change will be needed for a tag anyway

**Comment by skariel** [comment]
hi! i'm here

I'd be happy if someone wants to collaborate on this,
will take me some time otherwise

**Comment by tkelman** [comment]
Only thing needed from you would be merge https://github.com/JuliaGeometry/VoronoiDelaunay.jl/pull/20 if you approve of that change (at least the REQUIRE fix is necessary), then tag the package and submit a PR to METADATA. Anyone else with JuliaGeometry commit access should be able to do so.

**Comment by skariel** [comment]
merged. will tag later today. Need to read/remember how to do that

**Comment by robertdj** [comment]
Thanks for the access @SimonDanisch . I haven't tried organizations before. Should I move the package to `JuliaGeomerty` (once the release is ready)?

**Comment by SimonDanisch** [comment]
Yeah, would be great :)

**Comment by robertdj** [comment]
@skariel it doesn't look like you have published a new version/tag? 

It's described above this section in the docs: https://julia.readthedocs.io/en/release-0.4/manual/packages/#publishing-metadata-manually

**Comment by skariel** [comment]
done. see here: https://github.com/JuliaLang/METADATA.jl/pull/5836

**Comment by robertdj** [comment]
Thanks! Have you also committed a tag for GeometricalPredicates?

**Comment by skariel** [comment]
Yes - I just updated the PR

**Comment by skariel** [comment]
all merged. Closing this issue :)
